packages/block-editor-tools/src/hooks/use-debounce/index.js

```diff
@@ -10,6 +10,7 @@ import { useState, useEffect } from 'react';
  */
 const useDebounce = (value, delay) => {
   const [debouncedValue, setDebouncedValue] = useState(value);
+  console.warn('This custom useBounce hook has been deprecated. Please use the version from @wordpress/compose instead.');
```

Comment:
Let's add a notice here to ignore the unexpected console statement, since it is, in fact, expected.